### ISSUE TITLE
Problem with Learning Rate

### ISSUE THREAD

**Issue by stilllearningsoumya** [sayantan-kuila/pytorch#1]
How is the learning rate results so weird?

**Comment by sayantan-kuila** [comment]
when the learning rate is too low i.e in case when it is 0.00000000001 the testing and training loss is almost constant which means that the model is learning almost nothing.  the gradient updates are way too small to make any progress, or you can say that the model might get stuck in the local minima.
And when the learning rate is too high (0.1) then that means that the model is not reaching the global minima, it just toggles around that. so maybe that's why the model is diverging and the loss is increasing.
